lib/commands/exam.js

```diff
@@ -63,9 +63,13 @@ module.exports = TestCommand.extend({
     if (this.validator.shouldSplit) {
       commandOptions.query = addToQuery(commandOptions.query, '_split', commandOptions.split);
 
-      // Ignore the partition option when paralleling
+      // Ignore the partition option when paralleling (we'll fill it in later)
       if (!this.validator.shouldParallelize) {
-        commandOptions.query = addToQuery(commandOptions.query, '_partition', commandOptions.partition);
+        var partitions = commandOptions.partition;
+        if (partitions) {
+          partitions = partitions.join(',');
+        }
+        commandOptions.query = addToQuery(commandOptions.query, '_partitions', partitions);
```

Comment:
We should leave this as `_partition` for consistency with the CLI params. QUnit will automatically treat repeated URL params as an array. So if we do `tests?_partition=1&_partition=2` then you'll get `QUnit.urlParams._partition === ['1', '2']`.